docs/en/engines/database-engines/iceberg.md

```diff
@@ -0,0 +1,87 @@
+---
+slug: /en/engines/database-engines/iceberg
+sidebar_position: 30
+sidebar_label: Iceberg
+---
+import ExperimentalBadge from '@theme/badges/ExperimentalBadge';
+
+# Iceberg
+<ExperimentalBadge/>
+
+This database engine provides integration with external catalogs. The initial implementation supports Iceberg REST catalogs, with plans to extend support to other open table formats such as Delta Lake.
+
+:::note
+This is an experimental feature currently under development and may change in a backward-incompatible manner. You can enable it by setting:
+
+``` sql
+SET allow_experimental_database_iceberg=1;
+```
+:::
+
+## Creating a connection {#creating-a-connection}
+``` sql
+CREATE DATABASE catalog_name
+ENGINE = Iceberg(endpoint)
+[SETTINGS]
+[catalog_type = 'rest',]
+[catalog_credential = 'token',]  
+[warehouse = 'warehouse_name',]
+[oauth_server_uri = 'URL',]
+[auth_scope = 'all-apis,sql']
+```
+
+**Engine Parameters**
+
+- `endpoint`: Endpoint of the catalog
+
+
+## Using the catalog {#using-the-catalog}
+ClickHouse does not yet support an arbitrary number of namespaces. To avoid issues, enclose namespaces and table names in backticks when querying them. For example, if your table `my_table` is behind a catalog called `my_catalog` in the namespace `namespace1`, you should access it as follows:
+
+```sql
+SELECT * FROM my_catalog.`namespace1.my_table` LIMIT 10;
+```
+
+Once the connection is created, you can interact with tables in your catalog as if they were native ClickHouse tables.
+
+```sql
+SHOW TABLES FROM my_catalog;
+SHOW CREATE TABLE my_catalog.`namespace1.table_name`;
+SELECT * FROM my_catalog.`namespace1.my_table` LIMIT 10;
+```
+
+## Settings {#settings}
```

Comment:
Should we add default values?
You can check defaults here in the 3rd column https://github.com/ClickHouse/ClickHouse/blob/388882156f7e818bc2e894b28c7455d556af822b/src/Databases/Iceberg/DatabaseIcebergSettings.cpp#L17-L25
Also some settings are not covered: 
1. `vended_credentials`
2. `auth_header` (not sure though that this one would be useful for users, but it still exists) 
3. `storage_endpoint`. Would be useful in case non-aws implementation is used, for example `Minio`. But in case Catalog supports endpoints specification via config, then it will work without this setting after https://github.com/ClickHouse/ClickHouse/pull/75375 is merged. For example for Polaris catalog this support is not yet merged, only in an open PR now (https://github.com/apache/polaris/pull/389).

Reply:
Ok, will add that.